experimental/QuadFormAndIsom/src/enumeration.jl

```diff
@@ -669,7 +921,11 @@ function representatives_of_hermitian_type(G::ZZGenus, chi::Union{ZZPolyRingElem
     @hassert :ZZLatWithIsom 1 is_of_hermitian_type(MfM)
     first && return ZZLatWithIsom[MfM]
 
+    allow_info && println("Enumerate hermitian genus of rank $(rank(H))")
     gr = genus_representatives(H)
+    if is_definite(H)
+      @hassert :ZZLatWithIsom 1 mass(H) == sum(1//automorphism_group_order(HH) for HH in gr)
+    end
```

Comment:
We have observed some weird behavior regarding genus enumeration for hermitian lattices in Hecke. This is temporary until I have finished running the computations on a larger scale (to see if we catch further bugs/corner cases)